pgmax/bp/infer.py

```diff
@@ -97,7 +97,112 @@ def pass_fac_to_var_messages(
                 )
             )
         )
+    return ftov_msgs
+
+
+@jax.jit
+def pass_OR_fac_to_var_messages(
+    vtof_msgs: jnp.ndarray,
+    parents_states: jnp.ndarray,
+    children_states: jnp.ndarray,
+) -> jnp.ndarray:
+
+    """Passes messages from OR Factors to Variables.
+
+    Args:
+        vtof_msgs: Array of shape (num_edge_states,).
+            This holds all the flattened (binary) variables to factor messages.
+        parents_states: Array of shape (num_parents, 2)
+            parents_states[ii, 0] contains the global factor index
+            parents_states[ii, 1] contains the message index of the parent variable's state 0.
+            The parent variable's state 1 is parents_states[ii, 2] + 1
+        children_states: Array of shape (num_factors, 2)
```

Comment:
I don't think there should be such possibility. Still for consistency with the `parents_states` array, shouldn't we keep it that way?

Reply:
I think it would be better to avoid unnecessary redundancy. Also there's no obvious reason for having the parents consistent with the children, as they are two different kinds of variables

Reply:
fine with me